tests/testthat/test-combine-categories.R

```diff
@@ -169,4 +190,18 @@ with_test_authentication({
         expect_identical(names(categories(ds$combined_petloc)),
             c("Mammals", "Bird", "Skipped", "Not Asked"))
     })
+
+    test_that("collapseCategories works on categorical variable", {
+        ds$cat <- factor(sample(c("cat", "Cat", "dog", "aphid"), nrow(ds), replace = TRUE))
+        var <- ds$cat
```

Comment:
Don't do this--just refer to `ds$cat`. It's good to make sure that `ds$cat <- collapseCategories(ds$cat, ...)` works, that the object you're returning is acceptable for assignment back into the dataset.

Also, assert the category names before you do collapseCategories so that you can see how they are different after.